recipes/schema-salad/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "schema-salad" %}
+{% set version = "3.0.20181129082112" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 274d7dcc09f4d9fa364ac6f002e614354f4b554a4ca3a7c3dfa3002cd006e7f5
+
+build:
+  number: 0
+  entry_points:
+    - schema-salad-tool=schema_salad.main:main
+    - schema-salad-doc=schema_salad.makedoc:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - cachecontrol >=0.11.7
```

Comment:
Are all of these needed as host dependencies?

Reply:
For pure python packages host dependencies are dependencies needed to run the setup.py (so if your setup.py does something special (eg call pytest) then you need those things packages). Most pure python packages just have python and pip in their host section